app/code/Magento/GoogleGtag/Block/Ga.php

```diff
@@ -63,13 +70,16 @@ public function __construct(
         SerializerInterface $serializer,
         SearchCriteriaBuilder $searchCriteriaBuilder,
         OrderRepositoryInterface $orderRepository,
+        ProductRepositoryInterface $productRepository,
```

Comment:
I think we should make it null be default in accordance to backward compatibility.